protocol-definitions/Client.yaml

```diff
@@ -649,166 +649,33 @@ methods:
     name: statistics
     since: 2.0
     doc: |
-      The statistics is a String that is composed of key=value pairs separated by ',' . The following characters
```

Comment:
We need to add the compression/decompression algorithm doc here.

Reply:
Added the description of the compression algorithm.